documentation/modules/auxiliary/gather/snare_registry.md

```diff
@@ -0,0 +1,105 @@
+## Description
+
+  This module uses the Registry Dump feature of the [Snare Lite for Windows](https://sourceforge.net/projects/snare/) HTTP service on 6161/TCP to retrieve the Windows registry. The Dump Registry functionality is unavailable in Snare Enterprise.
+
+  Note: The Dump Registry functionality accepts only one connected client at a time. Requesting a large key/hive will cause the service to become unresponsive until the server completes the request.
+
+
+## Vulnerable Application
+
+  SNARE (System iNtrusion Analysis and Reporting Environment) is a series of log collection agents that facilitate centralised analysis of audit log data.
+
+  This module has been tested successfully with Snare Lite for Windows version 4.0.2.0 on Windows XP SP3.
+
+  Installers for Snare Lite are no longer available and the product is unsupported.
+
+
+## Verification Steps
+
+  1. Start `msfconsole`
+  2. Do: `use auxiliary/gather/snare_registry`
+  3. Do: `set rhost [IP]`
+  4. Do: `set HttpUsername [USERNAME]`
+  5. Do: `set HttpPassword [PASSWORD]`
+  6. Do: `run`
+  7. You should get a copy of *HKLM\HARDWARE\DESCRIPTION\System* from the remote Windows system
+
+
+## Options
+
+**HttpUsername**
+
+The username for Snare remote access (default: `snare`).
+
+**HttpPassword**
+
+The password for Snare remote access (default: blank).
+
+**REG_DUMP_KEY**
+
+Retrieve the specified registry key and all sub-keys.
+
+**REG_DUMP_ALL**
+
+Retrieve the entire Windows registry.
+
+**TIMEOUT**
+
+Timeout in seconds for downloading each registry key/hive.
+
+
+## Scenarios
+
```

Comment:
Lets add:
```
### Snare Lite for Windows version 4.0.2.0 on Windows XP SP3.
```

I like to put the versions down in the scenarios for clarity. You can keep or drop it from above in the vuln app section